app/javascript/src/components/Navigation/components/GlobalActivitiesModal.jsx

```diff
@@ -42,51 +42,108 @@ const StyledBottom = styled(Button)`
 
 const StyledModalBody = styled(Modal.Body)`
   background-color: ${COLOR_CONCRETE};
-  color: ${COLOR_MINE_SHAFT};;
+  color: ${COLOR_MINE_SHAFT};
 `;
 
-class GlobalActivitiesModal extends Component {
-  constructor(props) {
+type Props = {
+  more: boolean,
+  showModal: boolean,
+  onCloseModal: Function,
+  activites: Array<Activity>
+};
+
+type State = {
+  activities: Array<Activity>,
```

Comment:
I think `Activity` import is missing.

Reply:
is automatically imported by flow :smile:

Reply:
Nice :+1: :smiley: